### PR TITLE
Incorporate CRON expression settings in release manager checklist

### PR DESCRIPTION
**Pull request by alecharp**
Because we don't always want to do the release on Fridays and 11 AM UTC might not work for everyone, let's make setting the CRON expression for the build as the first item on the release manager TODO list.

### Testing done

For the scripts: I use them on my mac for the 2024-10-31 release.

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: release-manager-scripts/bom-line-test.sh [hunk 1]
```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+HERE="$(cd -P "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
```
Comment (alecharp) [inline release-manager-scripts/bom-line-test.sh:4]: This allows to call the scripts from anywhere and still being able to call colocated scripts.

**Comment by timja** [comment]
Can’t you trigger the build directly by re running the check using GitHub cli?

**Comment by alecharp** [comment]
I didn't try that @timja. 
The fact that the "button" didn't trigger anything led me to think we had something else going one. Changing the CRON was the quicker thing to do.

**Comment by timja** [comment]
> I didn't try that @timja. The fact that the "button" didn't trigger anything led me to think we had something else going one. Changing the CRON was the quicker thing to do.

the button is broken with the number of checks we have, but I would expect the CLI should work although haven't tested it

**Comment by alecharp** [comment]
I couldn't even start the first job run with the rerun from GitHub. So
before we have all the checks.

**Review by MarkEWaite — APPROVED** [review]
The changes look good to me.  My single comment is non-blocking.  Approved.

**Pull request merged**